### DISCUSSION TITLE
How to predict the host to use the appium:webDriverAgentUrl capability

### DISCUSSION THREAD

**Discussion by paulohvalipio** [appium/appium#19864] [Q&A]
I'm starting a WebDriverAgent with the command "xcodebuild -project ${diretorioWebDriverAgent}/WebDriverAgent.xcodeproj \
-scheme WebDriverAgentRunner \
-destination 'id=udid' test", but a WebDriverAgent starts on different hosts, I have no idea how to use the appium:webDriverAgentUrl capability. Would it be possible to predict the host or would there be another way to configure the capabilities to use the installed WebDriverAgent and start it?

Thanks!

**Comment by KazuCocoa** [comment]
What the "host" mean? Usually, the xcodebuild installs and start a session against `id=udid.` If the host meant different `udid` device, it is xcodebuild's issue I believe...
